### DISCUSSION TITLE
diff between importing from "zustand/react" and "zustand"

### DISCUSSION THREAD

**Discussion by leejss** [pmndrs/zustand#1339] [Q&A] [answered]
I'm using VScode.
When I type `create`, vscode automatically suggest `import create from "zustand/react";`

Are there differences between` import create from "zustand/react"` and `import create from "zustand"`?

**Comment by dai-shi** [comment, marked as answer]
It's a mistake by vscode. There's no such `zustand/react` thing. It's reported previously and I wonder what we can do to avoid vscode to get confused.
